beacon-chain/rpc/validator/assignments.go

```diff
@@ -149,7 +152,7 @@ func (vs *Server) duties(ctx context.Context, req *ethpb.DutiesRequest) (*ethpb.
 	}
 
 	return &ethpb.DutiesResponse{
-		Duties: validatorAssignments,
+		CurrentEpochDuties: validatorAssignments,
```

Comment:
`Duties` is now deprecated. Instead, we send out current epoch duties and next epoch duties